overlord/snapshotstate/snapshotstate_test.go

```diff
@@ -74,9 +76,34 @@ func (snapshotSuite) TestNewSnapshotSetID(c *check.C) {
 	c.Assert(err, check.IsNil)
 	c.Check(sid, check.Equals, uint64(1))
 
+	var stateSetID uint64
+	c.Assert(st.Get("last-snapshot-set-id", &stateSetID), check.IsNil)
+	c.Check(stateSetID, check.Equals, uint64(1))
+
+	c.Assert(ioutil.WriteFile(filepath.Join(dirs.SnapshotsDir, "9_some-snap-1.zip"), []byte{}, 0644), check.IsNil)
+
+	sid, err = snapshotstate.NewSnapshotSetID(st)
```

Comment:
each call NewSnapshotSetID to could use a short comment, like

// Disk last set id 9 >  state set id 1,  use 9++ = 10

etc

Reply:
Added.